### ISSUE TITLE
Fix .find to support `name`

### ISSUE THREAD

**Issue by vyper** [b2beauty/data_steroid#7]
O `.find` hoje suporta somente `id` (ou chave) do tipo numérico.

[Nesse](https://github.com/b2beauty/data_steroid/blob/master/lib/data_steroid/criteria/findable.rb#L9) trecho:

```ruby
run query.where('__key__', '=', gcloud_key(id.to_i, parent: parent))
```

Ele suporta somente `id` do tipo numérico, porque estamos forçando o `#to_i`, mas o correto deveria não interferir o no tipo do `id`.

**Comment by vyper** [comment]
@marcocarvalho @giovannibonetti :eyes:

**Comment by marcocarvalho** [comment]
Concordo

**Comment by giovannibonetti** [comment]
Concordo. Só depois vamos precisar testar tudo com cuidado pra evitar problemas por conta da mudança
